src/less/common.less

```diff
@@ -83,6 +59,11 @@ a, .pointer {
   }
 }
 
+//Container fluid
+.container-fluid.container-fluid-max-size{
+  max-width: 1392px;
```

Comment:
Try to override class values where possible instead of introducing new but arbitrary classes.

The following will limit the max width of `container-fluid` whilst keeping it stretchy even on extra large devices:

```less
.container-fluid {
  @media (min-width: @screen-lg-min) {
    width: @container-lg;
  }
  @media (min-width: @screen-xl-min) {
    width: @container-xl;
  }
}
```

Add this to `functions.less:172` to support a max width of our `container-fluid`'s and to extend the max size to `xl` screens.